recipes/tbb/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "4.4_20160128" %}
+{% set flat_version=version.replace(".", "") %}
+
+package:
+  name: tbb
+  version: {{ version }}
+
+source:
+  fn: tbb{{ flat_version }}oss_src_0.tgz
+  url: https://www.threadingbuildingblocks.org/sites/default/files/software_releases/source/tbb{{ flat_version }}oss_src_0.tgz
+  sha1: 80707e277f69d9b20eeebdd7a5f5331137868ce1
+
+build:
+  number: 0
+  skip: true  # [win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/tbb/tbb.h  # [unix]
+    - test -f ${PREFIX}/lib/libtbb.so  # [linux]
+    - test -f ${PREFIX}/lib/libtbb.dylib  # [osx]
+
+about:
+  home: http://www.threadingbuildingblocks.org
+  license: GPL v2
```

Comment:
According to the [licensing]( https://www.threadingbuildingblocks.org/licensing ), this is "GPLv2 with the (libstdc++) runtime exception". So could we please add something like that here? It makes a huge difference to note this. Not sure how we want to word it though. @ocefpaf @msarahan, care to comment?